### PR TITLE
NeuroDataReHack 2023

### PR DESCRIPTION
**Pull request by bendichter**
preview: https://neurodatawithoutborders.github.io/nwb_hackathons/pr-preview/pr-215/HCK16_2023_Granada_RH/

### CODE REVIEW

**Comment by bendichter** [comment]
preview: https://neurodatawithoutborders.github.io/nwb_hackathons/pr-preview/pr-215/HCK16_2023_Granada_RH/

**Review by oruebel — DISMISSED** [review]

File: HCK16_2023_Granada_RH/README.md [hunk 1]
```diff
@@ -1,77 +1,106 @@
...
-**Administrative Support:**
+## Logistics
+
+Thanks to the generous sponsorship of The Kavli Foundation, this event will be free to participants:
+* There is no registration or application fee.
+* Participants will be provided a private room at the Hotel Andalucia Center Hotel for the duration of this event, 
+  checking in on Monday, September 4 and checking out on Friday, September 8th, the day before the IBRO conference 
+  starts.
+* Breakfast, lunch, dinner, and coffee breaks will be provided.
+* You will be responsible for booking your own flight to the event. If you need financial assistance for travel,
```
Comment (rly) [inline HCK16_2023_Granada_RH/README.md:55]:
```suggestion
* You will be responsible for booking your own travel to the event. If you need financial assistance for travel, 
```

File: HCK16_2023_Granada_RH/README.md [hunk 2]
```diff
@@ -1,77 +1,106 @@
...
-In this workshop, we will teach attendees about the open neurophysiology datasets available on the DANDI Archive and
-train them on how to maximally utilize the archive and the NWB standard to incorporate existing data into their
-scientific workflows. This is the first event of its type for NWB, so our team will also be learning
-about how we can facilitate data reuse in the neuroscience community.
-
 ## Schedule
 
-TBD
+* Day 1, September 5: Searching the DANDI Archive, using the DANDI Hub, and reading NWB Files
+* Day 2, September 6: Tools to visualize and analyse NWB files
```
Comment (rly) [inline HCK16_2023_Granada_RH/README.md:81]:
```suggestion
* Day 2, September 6: Tools to visualize and analyze NWB files
```

File: HCK16_2023_Granada_RH/README.md [hunk 3]
```diff
@@ -1,77 +1,106 @@
...
+workshop, we will teach attendees about the open neurophysiology datasets available on the DANDI Archive and train 
+them on how to maximally utilize the archive and the NWB standard to incorporate existing data into their scientific 
+workflows. Feedback from attendees will be used to improve the software and data standard to better enable 
+reanalysis workflows.
 
-Apply to attend Neurodata Rehack 2 [here]
-<!--
-## Logistics
+Following the event, participants will be invited to apply for a Kavli Foundation Neurodata Discovery Award, which 
+awards $50,000 of funding to continue data reanalysis projects that come out of the NeuroDataReHack event.
```
Comment (rly) [inline HCK16_2023_Granada_RH/README.md:31]:
```suggestion
awards $50,000 (USD) of funding to continue data reanalysis projects that come out of the NeuroDataReHack event.
```

File: HCK16_2023_Granada_RH/README.md [hunk 4]
```diff
@@ -1,77 +1,106 @@
-# NeuroDataReHack 2
+# NeuroDataReHack 2023
```
Comment (rly) [inline HCK16_2023_Granada_RH/README.md:1]: Did you still want to rename the event Neurodata Rehack? Either way is fine with me.

File: HCK16_2023_Granada_RH/README.md (hunk 3 above)
Comment (rly) [inline HCK16_2023_Granada_RH/README.md:31]:
```suggestion
awards $50,000 of funding to continue data reanalysis projects that come out of the NeuroDataReHack event. Details about the Kavli Foundation Neurodata Discovery Award will be provided closer to the event. 
```

File: HCK16_2023_Granada_RH/README.md (hunk 4 above)
Comment (bendichter) [reply HCK16_2023_Granada_RH/README.md:1]: I've been going back and forth on it but I'm leaning towards keeping it as all camel case

File: HCK16_2023_Granada_RH/README.md [hunk 5]
```diff
@@ -1,77 +1,106 @@
...
 
-**Travel:**
+This event will primarily focus on analyzing existing data in NWB and on DANDI, **not** converting data to NWB. If 
+you are interested in learning how to convert data, consider signing up for an NWB User Days event.
 
--->
+## Application
 
-## Organizing Committee
+Space for the event is limited. Apply to attend NeuroDataReHack 2023 [here](https://forms.gle/akaThWFBJrv7udev9).
```
Comment (rly) [inline HCK16_2023_Granada_RH/README.md:42]: In the application, regarding the question "Please propose a potential project to derive scientific insight by analyzing one or more specific dandisets on the DANDI Archive (dandiarchive.org)." I recommend providing some guidelines about how long of a response we expect. A couple sentences? Max 200 words?

Comment (bendichter) [reply HCK16_2023_Granada_RH/README.md:42]: Good point. Let's say 250 words

**Review by rly — APPROVED** [review]

**Pull request merged**